expression/builtin_miscellaneous.go

```diff
@@ -306,45 +304,41 @@ func (c *inetNtoaFunctionClass) getFunction(args []Expression, ctx context.Conte
 	if err := c.verifyArgs(args); err != nil {
 		return nil, errors.Trace(err)
 	}
-	sig := &builtinInetNtoaSig{newBaseBuiltinFunc(args, ctx)}
+	bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpString, tpInt)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	bf.tp.Flen = 93
```

Comment:
Is it reasonable to use MySQL's 93 here? Since we know that at most there will be only 15 bytes.